action.yml

```diff
@@ -16,6 +16,10 @@ inputs:
     description: "Path of the directory containing your site"
     required: false
     default: "."
+  build-cmd:
+    description: "The command used to build your site"
+    required: false
+    default: "$PACKAGE_MANAGER run build"
```

Comment:
This assumes that the default build command is static and the same for all package managers, which is true _for now_ but might not be forever. A parallel PR adding Deno support would already break this assumption:

https://github.com/withastro/action/pull/77/files

I'd prefer to have this with a `default: ""` or some magic string and then check that if it isn't empty (or the magic string) use it, otherwise continues any logic following in the code. This would be more flexible.